clap.schema.json

```diff
@@ -69,10 +69,7 @@
           "type": "boolean"
         },
         "conflicts_with": {
-          "type": "string"
-        },
-        "conflicts_with_all": {
-          "$ref": "#/structures/arrayStrUnique"
+          "type": ["string", "#/structures/arrayStrUnique"]
```

Comment:
```suggestion
          "oneOf": [
            {
              "type": "string"
            },
            {
              "$ref": "#/structures/arrayStrUnique"
            }
          ]
```

Reply:
Actually, use `anyOf` instead of `oneOf`

Reply:
So it would be
```
"conflicts_with": {
  "anyOf": [
     {
       "type": "string"
     },
     {
       "$ref": "#/structures/arrayStrUnique"
     }
  ]
}
```

right? Just to be clear

Reply:
Yup, and then please squash all the commits into a single one.